packages/openapi2-parser/lib/generator.js

```diff
@@ -17,18 +17,22 @@ faker.option({
   random: () => 0,
 });
 
-const schemaIsArrayAndHasItems = schema => schema.type && schema.type === 'array' && schema.items;
+const schemaIsArrayAndHasItems = schema => schema.type === 'array' && typeof schema.items === 'object';
```

Comment:
`typeof schema.items === 'object'` is there a case where items wouldn't be an object? I think you'd get a parser error upfront when items is not an object:

> error: (4) Expected type object but found type array - line 31

Reply:
The intent here was to make sure the `schemaIsArrayAndHasItems` return value is true/false. Also further down the recursion we rely on the fact (access `items.examples` property) that it is an object.